include/cutlass/numeric_conversion.h

```diff
@@ -732,9 +732,14 @@ struct NumericConverterClamp<cutlass::half_t, S> {
   using source_type = S;
 
   CUTLASS_HOST_DEVICE
-    result_type operator()(source_type const &s) {
+    static result_type convert(source_type const & source) {
     return static_cast<cutlass::half_t>(s);
```

Comment:
`s` => `source`

Reply:
To clarify -- you'd like all of these function parameter definitions to be `source_type const &s`?